addons/membership/membership.py

```diff
@@ -136,6 +137,11 @@ def _state(self, cr, uid, ids, name, args, context=None):
             res[line.id] = state
         return res
 
+    def _set_state(self, cr, uid, ids, field_name, field_value, args, context=None):
+        cr.execute('UPDATE membership_membership_line '\
+                       'SET state=%s '\
+                       'WHERE id IN %s', (field_value, tuple([ids]),))
+        return True
```

Comment:
If the field is stored, the write should work. What's the purpose of the inverse function ? I agree those are necessary for non stored fields on which you want to enable a write. But this field is stored, so I don't understand the purpose of this modification.